pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -1319,6 +1319,13 @@ func (m *kubeGenericRuntimeManager) SyncPod(ctx context.Context, pod *v1.Pod, po
 	if isInPlacePodVerticalScalingAllowed(pod) {
 		if len(podContainerChanges.ContainersToUpdate) > 0 || podContainerChanges.UpdatePodResources {
 			m.doPodResizeAction(pod, podStatus, podContainerChanges, result)
```

Comment:
Thank you for updating. I'm afraid it turned out that no error is recorded into `result` (https://github.com/kubernetes/kubernetes/pull/125708#issuecomment-2250168367).  Please wait for #125708 to be merged.